.circleci/config.yml

```diff
@@ -83,6 +85,38 @@ jobs:
             # Slow, serial integration test, run nightly. Here to make sure the standard `mix test --only integration` works
             export SHELL=/bin/bash
             mix test --trace --only integration
+  barebuild_macos:
+    executor: metal_macos
+    environment:
+      MIX_ENV: test
+    steps:
+      - checkout
+      - run: echo 'export PATH=~/.cargo/bin:$PATH' >> $BASH_ENV
+      - run: |
+          brew install postgres
+          initdb /usr/local/var/postgres/data
+          pg_ctl -D /usr/local/var/postgres/data -l /tmp/postgresql.log start
+          psql template1 \<<EOF
+          CREATE USER omisego_dev WITH CREATEDB ENCRYPTED PASSWORD 'omisego_dev';
+          CREATE DATABASE omisego_dev OWNER 'omisego_dev';
+          EOF
+      - run: |
+          git clone https://github.com/asdf-vm/asdf.git ~/.asdf --branch v0.7.4
+          echo -e '\n. $HOME/.asdf/asdf.sh' >> ~/.bash_profile
+          echo -e '\n. $HOME/.asdf/completions/asdf.bash' >> ~/.bash_profile
+          source ~/.bash_profile
+          asdf plugin-add elixir https://github.com/asdf-vm/asdf-elixir.git
+          asdf plugin-add erlang https://github.com/asdf-vm/asdf-erlang.git
+          asdf install erlang 21.0
+          asdf install elixir 1.8.2
+          asdf global elixir 1.8.2
+          asdf global erlang 21.0
+      - run:
+          command: ./bin/setup
+          no_output_timeout: 2400
+      - run: make deps-elixir-omg
+      - run: ERLANG_ROCKSDB_BUILDOPTS='-j 2' make build-test
```

Comment:
`ERLANG_ROCKSDB_BUILDOPTS='-j 2'` was used to limit the amount of cores the compiling of rocksdb should use. the compiler used to detect the underlying host cores (which was 32) and that blew up. not sure how many cores there is in baremetal...